src/pynxtools/nomad/entrypoints.py

```diff
@@ -43,3 +47,110 @@ def load(self):
     mainfile_name_re=r".*\.nxs",
     mainfile_mime_re="application/x-hdf5",
 )
+
+apm_example = ExampleUploadEntryPoint(
+    title="Atom Probe Microscopy",
+    category="FAIRmat examples",
+    description="""
+        This example presents the capabilities of the NOMAD platform to store and standardize atom probe data.
+        It shows the generation of a NeXus file according to the
+        [NXapm](https://fairmat-nfdi.github.io/nexus_definitions/classes/contributed_definitions/NXapm.html#nxapm)
+        application definition and a successive analysis of an example data set.
+        The example contains a small atom probe dataset from an experiment with a LEAP instrument to get you started
+        and keep the size of your NOMAD installation small. Once started, we recommend changing the respective
+        input file in the NOMAD Oasis ELN to run the example with your own datasets.
+    """,
+    path="nomad/examples/apm",
```

Comment:
Instead of doing this, the best way would probably be to pull the examples from north directly. One can pass the `url` of a zip file to `ExampleUploadEntryPoint` (see [docs](https://nomad-lab.eu/prod/v1/staging/docs/reference/plugins.html#exampleuploadentrypoint)). Just need a way to zip the example folder from NORTH.